ui/js/controllers/perf/alerts.js

```diff
@@ -189,7 +189,7 @@ perf.controller('AlertsCtrl', [
                         !($scope.filterOptions.hideTo && alert.status === phAlertStatusMap.REASSIGNED.id &&
                           alert.related_summary_id !== alertSummary.id) &&
                         !($scope.filterOptions.hideTo && alert.status === phAlertStatusMap.DOWNSTREAM.id) &&
-                        (alert.status !== phAlertStatusMap.INVALID.id) &&
+                        !($scope.filterOptions.hideTo && alert.status == phAlertStatusMap.INVALID.id) &&
```

Comment:
At some point we may want to rename this `hideTo` variable that encompasses more of what it represents, but I don't think it's that important.